core/config.js

```diff
@@ -447,6 +449,23 @@ function getDefaultConfig() {
             }
         },
 
+        chatServers : {
+            mrc: {
+                enabled             : true,
+                serverHostname      : 'mrc.bottomlessabyss.net',
+                serverPort          : 5000,
+                multiplexerPort     : 5000,
+                bbsInfo : {
```

Comment:
You can get +op name from `StatLog.getSystemStat(SysProps.SysOpUsername)` or `StatLog.getSystemStat(SysProps.SysOpRealName)`

I wonder if some of those properties should be part of the general configuration along with `boardName` et. al.

Reply:
I think the `bbsInfo` stuff should be, yeah. I'll move it there and I'll change it if you hate it :)